fieldservice/models/fsm_location.py

```diff
@@ -65,6 +65,8 @@ class FSMLocation(models.Model):
                                group_expand='_read_group_stage_ids',
                                default=lambda self: self._default_stage_id())
 
+    notes = fields.Text(string="Notes")
+
```

Comment:
@osi-scampbell We already have a notes field on location:
https://github.com/OCA/field-service/blob/b4a901391bc7dc67a7f44c2933f221b922e39434/fieldservice/models/fsm_location.py#L48

Reply:
> @osi-scampbell We already have a notes field on location:
> https://github.com/OCA/field-service/blob/b4a901391bc7dc67a7f44c2933f221b922e39434/fieldservice/models/fsm_location.py#L48

I see, I don't see it on the view so I will leave the addition in the .xml but remove the duplicate in the .py